packages/ocurl/ocurl.0.7.6/opam

```diff
@@ -18,11 +18,12 @@ build-test: [
 ]
 build-doc: [[make "doc"]]
 remove: [["ocamlfind" "remove" "curl"]]
-depends: ["ocamlfind" {build}]
+depends: [
+  "ocamlfind" {build}
+  "conf-libcurl"
+]
 depopts: ["lwt"]
 depexts: [
-  [["debian"] ["libcurl4-gnutls-dev"]]
-  [["ubuntu"] ["libcurl4-gnutls-dev"]]
   [["centos"] ["libcurl-devel" "openssl-devel"]]
```

Comment:
I think that you should add the `centos` depexts to `conf-libcurl` to be able to remove all depexts here. The commit introducing the centos depexts is bc430e11f36925540fb7855c092d469eeea4a0a8 , and it quite clearly corresponds exactly to `libcurl4-gnutls-dev` depexts on other systems.